clang/lib/StaticAnalyzer/Checkers/MacOSKeychainAPIChecker.cpp

```diff
@@ -314,6 +314,7 @@ void MacOSKeychainAPIChecker::checkPreStmt(const CallExpr *CE,
     RegionArgIsBad = true;
   }
 
+  assert(ArgSM);
```

Comment:
I'm okay with this assert.
It shouldn't change much though because the very next lookup would fail if this was a null pointer because we never store entries for null pointers.
Thanks for the assert!